addons/mail/static/src/components/message/message.xml

```diff
@@ -146,14 +146,14 @@
                                     <t t-foreach="messageView.message.trackingValues" t-as="value" t-key="value.id">
                                         <li>
                                             <div class="o_Message_trackingValue">
-                                                <div class="o_Message_trackingValueFieldName o_Message_trackingValueItem" t-esc="value.changed_field"/>
-                                                <t t-if="value.old_value">
-                                                    <div class="o_Message_trackingValueOldValue o_Message_trackingValueItem" t-esc="value.old_value"/>
+                                                <div class="o_Message_trackingValueFieldName o_Message_trackingValueItem" t-esc="value.changedFieldAsString"/>
+                                                <t t-if="value.oldValueAsString">
+                                                    <div class="o_Message_trackingValueOldValue o_Message_trackingValueItem" t-esc="value.oldValueAsString"/>
                                                 </t>
                                                 <i t-else="" class="mr-2">None</i>
                                                 <div class="o_Message_trackingValueSeparator o_Message_trackingValueItem fa fa-long-arrow-right" title="Changed" role="img"/>
-                                                <t t-if="value.new_value">
-                                                    <div class="o_Message_trackingValueNewValue o_Message_trackingValueItem" t-esc="value.new_value"/>
+                                                <t t-if="value.newValueAsString">
+                                                    <div class="o_Message_trackingValueNewValue o_Message_trackingValueItem" t-esc="value.newValueAsString"/>
                                                 </t>
                                                 <i t-else="">None</i>
```

Comment:
Now that `MessageTrackingValue` is a model, could split this into its own component.
Should reduce complexity of `MessageComponent`.